corehq/ex-submodules/casexml/apps/case/xform.py

```diff
@@ -107,8 +107,8 @@ def attach_extras(case):
         mismatched_forms = action_xforms ^ set(case.xform_ids)
         if mismatched_forms:
             logging.warning(
-                "XFORM ID MISMATCH: action xform_ids don't match case xform_ids: "
-                "case_id: {}".format(
+                "CASE XFORM MISMATCH /a/{},{}".format(
+                    domain,
```

Comment:
@czue Couchlog automatically parses out the domain from the message and
then I can easily split the message on the comma to get the case ID.